.travis.yml

```diff
@@ -9,32 +9,32 @@ matrix:
   include:
     # A. Mac OSX
     - os: osx
-      osx_image: xcode9.4
+      osx_image: xcode10.2
```

Comment:
What does that have to do with a license change?

Reply:
I roll back the modifications. It appears only in this branch, but it is general now.